spring-integration-core/src/main/java/org/springframework/integration/config/xml/PointToPointChannelParser.java

```diff
@@ -30,6 +28,7 @@
 import org.springframework.integration.store.MessageGroupQueue;
 import org.springframework.util.StringUtils;
 import org.springframework.util.xml.DomUtils;
+import org.w3c.dom.Element;
```

Comment:
Hi @artembilan 

>So, this one imports order change is wrong

I see, but just I did copy from other classes... Seems the problem comes from
[Removing whitespace...](http://stackoverflow.com/a/2618521)